### PR TITLE
alert mechanism

### PR DESCRIPTION
**Pull request by pnuckowski**
* send trigger/resolve events

### CODE REVIEW

File: alamo_alerter/alerter.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+import json
+import time
+from datetime import datetime
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: ordering ;)

File: alamo_alerter/alerter.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        self.session.auth = ()
+        self.session.headers.update({
+            'content-type': 'application/json'
+        })
+
+    def rebuild_cache(self, debounce: int) -> deque:
+        """Based on Cassandra returned value rebuild results.
+        rebuild form DAO...
+
+        :param debounce:
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: describe debounce please

File: alamo_alerter/cache.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```
Comment (deejay1) [inline alamo_alerter/cache.py]: aren't we running that on py3.4 anyway?

File: README.rst [hunk 4]
```diff
@@ -10,11 +10,37 @@ Alamo alerter
 
 
 Alerter service for ALAMO project.
+Main goal of this software is to analyze check results and based on those result
```
Comment (h00dy) [inline README.rst]: based on those <b>results</b>

File: README.rst [hunk 5]
```diff
@@ -10,11 +10,37 @@ Alamo alerter
...
 
-Features
---------
+Information
+-----------
+* Alerter is actually customized rq worker class http://python-rq.org/docs/workers/.
+    Alerter requires a bit of information to work. Those information are taken from config file.
+    By default it reads from `config.cfg` file,
+    but could be overwritten by setting environment variable `ALAMO_ALERTER_CONFIG`.
+    Configuration form config file are passed to **EACH** executed job.
```
Comment (h00dy) [inline README.rst]: Configuration <b>from</b> config

File: alamo_alerter/alerter.py [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+        self.session = Session()
+        self.session.auth = ()
+        self.session.headers.update({
+            'content-type': 'application/json'
+        })
+
+    def rebuild_cache(self, debounce: int) -> deque:
+        """Based on Cassandra returned value rebuild results.
+        rebuild form DAO...
```
Comment (h00dy) [inline alamo_alerter/alerter.py]: form? or from?

File: tests/test_alerter.py [hunk 7]
```diff
@@ -0,0 +1,121 @@
+from __future__ import unicode_literals
+# -*- coding: utf-8 -*-
```
Comment (deejay1) [inline tests/test_alerter.py]: still wrong order ;)

**Comment by deejay1** [comment]
:+1:

**Comment by ch40s3k** [comment]
:+1:

**Pull request merged**